moveit_ros/occupancy_map_monitor/src/occupancy_map_server.cpp

```diff
@@ -75,8 +75,8 @@ int main(int argc, char** argv)
 {
   rclcpp::init(argc, argv);
   auto node = rclcpp::Node::make_shared("occupancy_map_server");
-  auto octree_binary_pub =
-      node->create_publisher<octomap_msgs::msg::Octomap>("octomap_binary", RMW_QOS_POLICY_HISTORY_KEEP_LAST);
+  rclcpp::QoS qos(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_default));
```

Comment:
Doesn't this have the same effect as leaving `qos` empty?

Reply:
Dusting this very old PR off, I'm not sure why I made this change. Yes, it should have the same effect as leaving it empty, but I don't remember what my motivation was for changing this to default (which is reliable, instead of best effort). I think this was under the assumption that single octomap "snapshots" were going to be sent for what I was trying. But I don't think that is a good assumption to merge into main and I should revert this